esp/src/eclwatch/stub.js

```diff
@@ -40,13 +41,14 @@ define([
 
 ], function (fx, dom, domStyle, ioQuery, ready, lang, arrayUtil, topic, xhr, cookie,
             Dialog, Button,
-            ESPUtil, Utility,
+            ESPUtil, Utility, LockDialogWidget,
             entities, Toaster) {
 
-    var IDLE_TIMEOUT = cookie("ESPSessionTimeoutSeconds") * 1000  || 7200 * 1000;
+    var IDLE_TIMEOUT = cookie("ESPSessionTimeoutSeconds") * 1000;
```

Comment:
Does this need a default value if no cookie exists?

Reply:
No because with the new config manager fix @kunalaswani reported and @wangkx || @RussWhitehead  is implementing the cookie is set with a minimum time. I also only use this variable is session management exists (cookie("ESPSessionTimeoutSeconds"));